dockerfiles/pyopenms/manylinux/Dockerfile

```diff
@@ -1,38 +1,39 @@
 FROM quay.io/pypa/manylinux2014_x86_64
-
-#defaults, can be overwritten from outside
-ARG OPENMS_BRANCH="master" 
+ARG CONTRIB_BUILD_DIR="/contrib-build"
 ARG OPENMS_VERSION="latest"
 
 # make source files from context available in docker
 COPY . /contrib
 
-RUN yum install -y wget
-RUN yum install -y xz qt5-qtbase-devel 
-RUN yum install -y qt5-qtbase-devel qt5-qtsvg qt5-qtsvg-devel python-devel 
+RUN yum install -y \
+    wget \
+    xz \
+    qt5-qtbase-devel \
+    qt5-qtsvg-devel \
+    python-devel \
+    qt5-qtsvg \
+    libtool \
+    cmake3
 # RUN yum install -y libsvm-devel glpk-devel libzip-devel zlib-devel xerces-c-devel bzip2-devel sqlite-devel hdf5-devel
```

Comment:
Yes, apparently we tried that before.
Until recently CoinOR (fixed) and Wildmagick (removed) were problems.
We can try CoinOR from yum if we find it.